lib/pubnub/client/getters_setters.rb

```diff
@@ -115,19 +117,19 @@ def timetoken=(timetoken)
         @env[:timetoken] = timetoken
       end
 
-      #   <b>DEPRECATED:</b> Please use <tt>user_id</tt> instead.
       # Returns:
       # ========
       # Current uuid.
       def uuid
-        @env[:uuid]
+        user_id
       end
+      deprecate :uuid, :user_id, 2023, 1
 
       # Returns:
       # ========
       # Current user_id.
       def user_id
-        uuid
+        @env[:uuid]
```

Comment:
@kleewho why not suggested `@env[:user_id]`?
`env` hash used internally, lets change names now, so it future we won't think why only some of them has been changed.

Reply:
I wasn't sure if it's used or not by our users.